README.md

```
@@ -1,95 +1,39 @@
-IntelliJ에서 Spring Boot를 `local` 프로파일로 실행하려면 Run Configuration을 아래와 같이 설정합니다.
+### 실행
 
-1. 상단 메뉴 **Run → Edit Configurations** 선택
-2. `ReadumApplication` 실행 구성 선택 (없으면 `+` → `Spring Boot` 추가)
-3. 아래 항목을 설정합니다.
+**Active profiles:** `dev`
 
-**Active profiles**
+IntelliJ Run Configuration에서 환경 변수를 설정합니다.
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)